typescript/nomad-deploy/scripts/twoLocals/connect.ts

```diff
@@ -0,0 +1,82 @@
+import * as tom from '../../config/local/tom';
+import * as daffy from '../../config/local/jerry';
+import { ExistingCoreDeploy } from '../../src/core/CoreDeploy';
+import { deployEnvironment } from '../../src/chain';
+import { ExistingBridgeDeploy } from '../../src/bridge/BridgeDeploy';
+import { enrollSpoke } from '../../src/incremental';
+import { NomadContext } from '@nomad-xyz/sdk';
+
+let environment = deployEnvironment();
+
+const path =
```

Comment:
does the local env ever use staging configs? should we simplify this to only ever use dev configs?

Reply:
https://github.com/nomad-xyz/nomad-monorepo/pull/17/commits/ff75273178b6c024dbce8b40399362d8e5c5d680